### PR TITLE
docs: lead

### PR DESCRIPTION
**Pull request by ableksaju**

### CODE REVIEW

**Comment by ableksaju** [comment]
@Akshayitzme please review

File: docs/entries/lead.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+A Lead can have various statuses based on the stage of engagement:
+
+|   # | Status         | Description                                                      |
+| --: | :------------- | :--------------------------------------------------------------- |
+|   1 | Open           | Newly created and yet to be engaged.                     |
+|   2 | Replied        | Replied to initial engagement.                          |
+|   3 | Interested     | Shown interest in the product or service.               |
+|   4 | Opportunity    | Opportunity for business.                  |
+|   5 | Converted      | Converted to a customer or client.                 |
```
Comment (akshayitzme) [inline docs/entries/lead.md]: replace `customer` to `Customer`.

Frappe Books specific words should start with caps

File: docs/entries/lead.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+|   7 | Do not Contact | Marked as not to be contacted.                     |
+
+::: warning Usage
+Ensure to update the status of the Lead appropriately as it progresses through different stages.
+:::
+
+## Lead Fields
+
+1. **Name**: The full name of the Lead. This is a required field to uniquely identify each Lead.
+2. **Status**: The current status of the Lead. It indicates the stage of engagement with the Lead and can be one of the following:
```
Comment (akshayitzme) [inline docs/entries/lead.md]: Incomplete sentence, 
> Lead and can be one of the following:

File: docs/entries/images/lead.png
Comment (akshayitzme) [inline docs/entries/images/lead.png:1]: It would be better if you create a Lead document with a legitimate name

File: docs/entries/lead.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+The Lead Quick Edit Form includes actions to create associated documents like Customer or Sales Quote.
+
+![Recent Actions Widget](./images/lead.png)
+
+### Create Customer
+
+- Converts the Lead into a Customer, retaining all relevant information.
+- Sets the status of the Lead to 'Converted'.
+
```
Comment (akshayitzme) [inline docs/entries/lead.md]: 'Converted' can be bolded, quotes can be removed

File: docs/entries/lead.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+### Create Sales Quote
+
+- Creates a Sales Quote for the Lead.
```
Comment (akshayitzme) [inline docs/entries/lead.md]: ❌  for the Lead
✅  against the Lead

**Pull request closed without merging**